### ISSUE TITLE
Regarding the issue of 'unsupervised_methods' not being found

### ISSUE THREAD

**Issue by daiqingyue** [ziiho08/Periodic-MAE#2]
Hello, I was preparing to run your program, but I encountered the following problem: from unsupervised_methods.methods import POS_WANGModuleNotFoundError: No module named 'unsupervised_methods'Did you perhaps not make 'unsupervised_methods' publicly available in the GitHub project? I would appreciate your clarification, thank you!

**Comment by daiqingyue** [comment]
I found unsupervised_methods in rPPG-Toolbox, but a new error occurred: File "/home/Periodic-MAE-main/Periodic-MAE-main/neural_methods/loss/TorchLossComputer.py", line 12, in <module>from unsupervised_methods.methods.POS_WANG import torch_detrending_filter, torch_butter_bandpass_filterImportError: cannot import name 'torch_detrending_filter' from 'unsupervised_methods.methods.POS_WANG' (/home/dai/Project_phd/Periodic-MAE-main/Periodic-MAE-main/unsupervised_methods/methods/POS_WANG.py). The POS_WANG.py in rPPG-Toolbox does not provide the packages torch_detrending_filter and torch_butter_bandpass_filter.
